portfolio/src/main/webapp/script.js

```diff
@@ -12,29 +12,23 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Computer Science', 'Business', 'Design', 'Marketing'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+function getFacts() {
+  fetch('/data').then(response => response.json()).then((fun_facts) => {
+    
```

Comment:
Use camelCase for variables. So funFacts instead of fun_facts

Reply:
Done!